cmd/svcat/plan/describe_cmd.go

```diff
@@ -95,28 +97,31 @@ func (c *describeCmd) describe() error {
 		}
 		plan, err = c.App.RetrievePlanByClassAndPlanNames(names[0], names[1])
 	} else {
-		plan, err = c.App.RetrievePlanByName(c.name)
+		plan, err = c.App.RetrievePlanByName(c.name, servicecatalog.ScopeOptions{
+			Scope: servicecatalog.ClusterScope,
+		})
 	}
 	if err != nil {
 		return err
 	}
 
+	csp := plan.(*v1beta1.ClusterServicePlan)
```

Comment:
We can't safely convert to a specific type here (ClusterServicePlan) because the `svcat describe plan` command needs to work with both types that implement the Plan interface. 

Would you please leave a `// TODO: This convert will be removed and implemented properly in https://github.com/kubernetes-incubator/service-catalog/issues/2216`. Just so that everyone knows why we are doing that unsafe convert right now.